### DISCUSSION TITLE
Cannot Reize App Window??

### DISCUSSION THREAD

**Discussion by drghty** [Y0URD34TH/Project-GLD#103] [Q&A] [answered]
Hi. I'm unsure if this is a bug, but I cannot resize the app window. Is this intentional? I don't see why it would be.

Cheers

**Comment by Y0URD34TH** [comment, marked as answer]
hey imgui indeed support an resize, but we did majority of things in imgui and directx 9 with static positioning (it was more fast to we do taht way), we can change it but it would take an lot of time to make an resize function soo maybe in future itll have.
(positioning and size in imgui and dx9 is kinda hard and annoying xD)
